test/sagemaker_tests/pytorch/training/conftest.py

```diff
@@ -30,13 +30,10 @@
 from . import get_efa_test_instance_type
 
 from .utils import get_ecr_registry, NightlyFeatureLabel, is_nightly_context
-from .integration import (
-    get_framework_and_version_from_tag,
-    get_cuda_version_from_tag,
-    get_processor_from_image_uri,
```

Comment:
Is this not being used in this `conftest` anymore?

Reply:
This function was only being used in `skip_p5_tests()`. I replaced it with the `processor` fixture.